src/components/macro/macro-action-editor/tab/macro-key/macro-key.component.ts

```diff
@@ -0,0 +1,73 @@
+import { KeyAction } from '../../../../../config-serializer/config-items/key-action/KeyAction';
+import { EditableMacroAction } from '../../../../../config-serializer/config-items/macro-action/EditableMacroAction';
+import { MacroSubAction } from '../../../../../config-serializer/config-items/macro-action/MacroAction';
+import { KeypressTabComponent } from '../../../../popover/tab/keypress';
+import { Tab } from '../../../../popover/tab/tab';
+import { Component, Input, OnInit, ViewChild } from '@angular/core';
```

Comment:
tslint seemed to regard the .. stuff as preceding the angular stuff. That's why I was against the imports ordering rule because it makes for less logical reading.